riklet/src/runtime/function_runtime.rs

```diff
@@ -79,9 +79,10 @@ impl FunctionRuntime {
             )
             .try_build()
             .map_err(RuntimeError::FirepilotConfiguration)?;
-        let executor = FirecrackerExecutorBuilder::new()
+
+        let executor = FirecrackerExecutorBuilder::auto()
+            .map_err(RuntimeError::FirepilotConfiguration)?
             .with_chroot(DEFAULT_FIRECRACKER_WORKSPACE.to_string())
-            .with_exec_binary(self.function_config.firecracker_location.clone())
```

Comment:
Even though we can autoload the binary, we might have cases where we can't determine where the binary is, we should document how we can override through environment variables (see [firepilot](https://github.com/rik-org/firepilot/blob/main/firepilot/src/builder/executor.rs#L86)).

Might make sense to create a `riklet` page in the documentation for that

Reply:
We could also create an hint when there is an error auto-loading, to tell people to use the environment variable.

Reply:
For the doc i have had [that](https://github.com/rik-org/rik/pull/123/files#diff-3de7d05e1030ee76f5f6d1bed7ab301cf54f4669e2e44e9773ea82093dab20caR63) did you see it ?

Reply:
> We could also create an hint when there is an error auto-loading, to tell people to use the environment variable.

I think the better is to add this hint into firepilot directly not in rik

Reply:
@AlexandreBrg

Reply:
Published in firepilot `1.1.1`